nursery/linked-against-libsodium.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: linked against libsodium
+    namespace: linking/static/crypto
+    authors:
+      - "@mr-tz"
+    scope: file
+    mbc:
+      - Cryptography::Crypto Library [C0059]
```

Comment:
It would be useful to add a `description` meta with a short libsodium description.